app/[domain]/page.tsx

```diff
@@ -202,22 +210,73 @@ export default async function IndexPage({
         >
           <p className="max-w-lg text-sm">
             Once you have submitted your chosen handle above, it will be created
-            and you can now change it in the Bluesky app.<br/>
-            Go to Settings {">"} Advanced {">"} Change my handle.<br/> 
+            and you can now change it in the Bluesky app.
+            <br />
+            Go to Settings {">"} Advanced {">"} Change my handle.
+            <br />
             Select &quot;I have my own domain&quot; and enter{" "}
-            {newHandle ? `"${newHandle}"` : "your new handle"}.<br/>
-            Leave the setting on DNS Panel and ignore the text box as this is all set up automatically.<br/>
+            {newHandle ? `"${newHandle}"` : "your new handle"}.<br />
+            Leave the setting on DNS Panel and ignore the text box as this is
+            all set up automatically.
+            <br />
             Finally, tap &quot;Verify DNS Record&quot;.
           </p>
+        </Stage>
+        <div className="max-w-lg text-sm">
           <p className="mt-6 max-w-lg text-sm">
-            This project was initially created by mozzius, and adapted by ChrisRid for the NAFO fellas.<br/>
-            If you like mozzius project, consider{" "}
-            <a href="https://github.com/sponsors/mozzius" className="underline">
-              sponsoring their work.
-            </a>
-            .
+            This Service is made for the NAFO fellas on Bluesky Visit there
+            official website and help support them.
           </p>
-        </Stage>
+          <div className="py-4">
+            <Image
+              src="/images/NAFO.png"
+              width={512}
+              height={400}
+              alt="NAFO image"
+            />
+          </div>
+          <div className="max-w-lg text-center">
+            <p className="text-sm">
+              To help support the hosting costs for this service:
+            </p>
+            <p className="py-2 text-center">
+              <a
+                href="https://buymeacoffee.com/chrisrid"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                buymeacoffee.com/chrisrid
+              </a>
+            </p>
+            <p>
+              (Please prioritize supoorting Ukrainan charities / NAFO first)
```

Comment:
Typo - "supoorting" should be "supporting"
Type - "Ukrainan" should be "Ukrainian"